packages/tools/devtools/devtools-core/package.json

```diff
@@ -11,20 +11,62 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
Shouldn't these be the public rollups? Or is this an intermediate step to maintain current behavior?

Reply:
Changing the `.` exports would be a breaking change. So I can see the benefit in doing that separately. But updating the others to point to the appropriate rollups seems safe.